### PR TITLE
Update dependencies in Beta

### PR DESCRIPTION
**Pull request by pradeepnschrodinger**
## Description

Redone all the steps for updating the dependencies

* #434 - Initial update
* #440 - Update PostCSS
* #450 - Initial babel upgrade
* #451 - Calypso codemods
* #452 - Fix site and tests
* #453 - Fix internal build

## Motivation and Context

Fixes [SS-25810](https://jira.schrodinger.com/browse/SS-25810)

## How Has This Been Tested?

Tested on LD and site examples.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by pradeepnschrodinger** [comment]
The tests seem to be failing meh 🤔 
I think we'll have to apply the codemods on the test folder.

**Comment by pradeepnschrodinger** [comment]
> lgtm. Thanks for slogging through these changes.
> 
> Let me know if anything novel comes up while modifying the tests.

Thanks, got the tests working by 97429c5369babfb37a4a2ffe867972bde579648f. What happened was that a test file was using a default styled import over FixedDataTableRoot.js
But it doesn't use a default export.

I also removed `updtr` and `calypso-codemods` since they're no longer needed.

I'll keep this review open for 1 more day before merging and cutting a release.

**Comment by wcjordan** [comment]
Great, thanks!

**Review by Az111z2 — APPROVED** [review]

File: examples/FilterExample.js [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 "use strict";
```
Comment (Az111z2) [inline examples/FilterExample.js:5]: a

**Pull request merged**